src/utils/validateEmailAddress.test.tsx

```diff
@@ -7,11 +7,13 @@ describe("Email validation", () => {
     "test2@gmail.com.sg",
     "      test3@example.com",
     " test4@example.com     ",
+    " test5@example.co.sg",
+    "test-6@example.com",
```

Comment:
Could you add a test case for `test+a1@example.com` too?

Reply:
added